versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.0"
+      version = ">= 5.61.0"
```

Comment:
Nit: I don't think you need this MSV bump (the plugin cache was the issue but is now resolved)

Reply:
got it... should I revert or leave it?